cdisc_rules_engine/check_operators/dataframe_operators.py

```diff
@@ -1048,16 +1048,26 @@ def non_empty_within_except_last_row(self, other_value: dict):
     @type_operator(FIELD_DATAFRAME)
     def contains_all(self, other_value: dict):
         target = self.replace_prefix(other_value.get("target"))
+        value_is_literal: bool = other_value.get("value_is_literal", False)
         comparator = other_value.get("comparator")
-        if isinstance(comparator, list):
-            # get column as array of values
-            values = flatten_list(self.value, comparator)
+        if self.is_column_of_iterables(
```

Comment:
documentation should be updated for `contains_all` to explain that this will work with:
- a column of literals
- a column of lists
- a list of literals

(if this statement is true)